v2src/template/autotune_table_entry.cc

```diff
@@ -5,6 +5,7 @@
 #include <aotriton/_internal/triton_kernel.h>
 #include <aotriton/_internal/kernel_cluster.h>
 #include <aotriton/cpp_tune.h>
+#include <iostream>
```

Comment:
Need to guard this with `#ifndef NDEBUG`-`#endif`

This was removed due to extra static initialization overhead by pulling into this header file.
The content of this template file will be duplicated into about 9000+ files (depending on the number of arch to support) and any small overheads will eventually be significant.

Otherwise LTGM